materials/databases_intro.md

```diff
@@ -5,75 +5,79 @@ title: Introduction to Databases
 language: R
 ---
 
-**If you haven't already install Firefox**
+> Remind students to install Firefox
 
-**Walk through installing SQLite Manager add-on**
+> Walk through installing SQLite Manager Add-on
 
 ## Why use a database management system
 
-* Designed for storing, managing, and retrieving information
-* Lots of existing data is stored in them
-* Fast for large amounts of data
-* Out of memory processing for really large data
-* Improve quality control of data entry (type constraints and use of forms in
-  Access, Filemaker, etc.)
-* Can handle a spatially explicit queries (GIS)
-* The concepts of relational database querying are core to understanding how to
-  do similar things using programming languages such as R or Python.
+* Simple and effective software for storing, managing and retrieving information
+* Widely used to store and maintain lots of existing data
+* Fast processing across large amounts of data
+    * Cloud-based processing for really large data
+    * Can handle spatially explicit queries (GIS)
+* Improve quality control of data entry using scripts and tests
+    * vs. type constraints and use of forms in Access, Filemaker, etc.
```

Comment:
Not sure what you are trying to say here about 'Improve quality control of data entry'.  I think clarifying this item could help link or motivate Exercise 1 - 'Data Validation in Excel'.